internal/repository/prisma/dbsqlc/step_runs.sql

```diff
@@ -340,11 +340,11 @@ step_runs AS (
         sr."tenantId" = @tenantId::uuid
         AND ((
             sr."status" = 'RUNNING'
-            AND w."lastHeartbeatAt" < NOW() - INTERVAL '60 seconds'
+            AND w."lastHeartbeatAt" < NOW() - INTERVAL '30 seconds'
```

Comment:
For consistency, if we are enforcing 30 seconds as a global worker heartbeat timeout, then we should fail steps with no retries left which belong to a timed-out worker. So shouldn't we remove `AND s."retries" > sr."retryCount"`?

Reply:
good catch